tests/data/hrrr/nomads_response.html

```diff
@@ -0,0 +1,35 @@
+
+<body>
+    <h1>Index of /pub/data/nccf/com/hrrr/prod/hrrr.20210614/conus</h1>
+    <pre>Name                                        Last modified      Size
+                    <hr><a href="/pub/data/nccf/com/hrrr/prod/hrrr.20210614/">Parent Directory</a>                                                 -   
+                    <a href="bufrsnd.t10z/">bufrsnd.t10z/</a>                               14-Jun-2021 01:53    -   
+                    <a href="bufrsnd.t01z/">bufrsnd.t01z/</a>                               14-Jun-2021 02:29    -   
+                    <a href="hrrr.t10z.bufrsnd.tar.gz">hrrr.t10z.bufrsnd.tar.gz</a>                    14-Jun-2021 01:54   41M  
+                    <a href="hrrr.t10z.class1.bufr.tm00">hrrr.t10z.class1.bufr.tm00</a>                  14-Jun-2021 01:52   89M  
+                    <a href="hrrr.t10z.wrfnatf00.grib2">hrrr.t10z.wrfnatf00.grib2</a>                   14-Jun-2021 00:52  652M  
+                    <a href="hrrr.t10z.wrfnatf00.grib2.idx">hrrr.t10z.wrfnatf00.grib2.idx</a>               14-Jun-2021 00:52   60K  
+                    <a href="hrrr.t10z.wrfnatf01.grib2">hrrr.t10z.wrfnatf01.grib2</a>                   14-Jun-2021 00:53  677M  
+                    <a href="hrrr.t10z.wrfnatf01.grib2.idx">hrrr.t10z.wrfnatf01.grib2.idx</a>               14-Jun-2021 00:53   68K   
+                    <a href="hrrr.t10z.wrfprsf00.grib2">hrrr.t10z.wrfprsf00.grib2</a>                   14-Jun-2021 00:52  386M  
+                    <a href="hrrr.t10z.wrfprsf00.grib2.idx">hrrr.t10z.wrfprsf00.grib2.idx</a>               14-Jun-2021 00:52   32K  
+                    <a href="hrrr.t10z.wrfprsf01.grib2">hrrr.t10z.wrfprsf01.grib2</a>                   14-Jun-2021 00:53  403M  
+                    <a href="hrrr.t10z.wrfprsf01.grib2.idx">hrrr.t10z.wrfprsf01.grib2.idx</a>               14-Jun-2021 00:53   38K  
+                    <a href="hrrr.t10z.wrfsfcf00.grib2">hrrr.t10z.wrfsfcf00.grib2</a>                   14-Jun-2021 00:52  134M  
+                    <a href="hrrr.t10z.wrfsfcf00.grib2.idx">hrrr.t10z.wrfsfcf00.grib2.idx</a>               14-Jun-2021 00:52  9.3K  
+                    <a href="hrrr.t10z.wrfsfcf01.grib2">hrrr.t10z.wrfsfcf01.grib2</a>                   14-Jun-2021 00:53  141M  
+                    <a href="hrrr.t10z.wrfsfcf01.grib2.idx">hrrr.t10z.wrfsfcf01.grib2.idx</a>               14-Jun-2021 00:53   10K  
+                    <a href="hrrr.t10z.wrfsubhf00.grib2">hrrr.t10z.wrfsubhf00.grib2</a>                  14-Jun-2021 00:51   48M  
+                    <a href="hrrr.t10z.wrfsubhf00.grib2.idx">hrrr.t10z.wrfsubhf00.grib2.idx</a>              14-Jun-2021 00:51  2.5K  
+                    <a href="hrrr.t10z.wrfsubhf01.grib2">hrrr.t10z.wrfsubhf01.grib2</a>                  14-Jun-2021 00:52  186M  
+                    <a href="hrrr.t10z.wrfsubhf01.grib2.idx">hrrr.t10z.wrfsubhf01.grib2.idx</a>              14-Jun-2021 00:52   11K  
+                    <hr></pre>
+</body>
+
+</html>
```

Comment:
How about putting this file into a dedicated 'NOMADS' folder under `tests`?

This would separate test logic from test data and feels then similar on how we keep the RME data organized.

Reply:
Moved to it's own folder `tests/nomads`